authors/apps/articles/tests/test_create_article.py

```diff
@@ -41,3 +43,25 @@ def test_api_cannot_create_an_article_with_an_unauthenticated_user(self):
         )
 
         self.assertEqual(response.status_code, status.HTTP_403_FORBIDDEN)
+
+    def test_that_every_256_words_or_below_is_1_min_read(self):
+        article = self.data
+
+        response = self.client.post(
+            '/api/articles/',
+            {"article": article},
+            format="json"
+        )
+
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+        self.assertIn('1 min read', response.data["read_time"])
+
+    def test_that_a_word_over_256_words_increases_the_readtime(self):
+        response = self.client.post(
+            '/api/articles/',
+            {"article": valid_article_2},
+            format="json"
+        )
+
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+        self.assertIn('2 min read', response.data["read_time"])
```

Comment:
@richien, @NicholusMuwonge, @Manorlds-Eaglespark and @reifred,
I believe we need a check such that builds will fail if we do not follow standards.
Kindly look into how to use flake8 0r pep8 so PR tests do not pass if we have issues such as the missing line here.

Reply:
Ok @RonKbS, I am working on adding flake8 to the branch I am currently working on.

Reply:
I applaud your proactiveness in doing so @richien. But we will have it as a separate story for next week. For now let us go ahead without them. Same goes for you, @NicholusMuwonge. Please remove the flake8 implementation from this PR, as well as the changes I see related to creating profiles
<img width="1066" alt="Screenshot 2019-04-11 at 14 15 37" src="https://user-images.githubusercontent.com/40039818/55953255-4d026b80-5c64-11e9-86c0-31d29214b333.png">

Reply:
I did remove the implementation